python/pyspark/sql/functions.py

```diff
@@ -793,8 +824,8 @@ def date_format(date, format):
     .. note:: Use when ever possible specialized functions like `year`. These benefit from a
         specialized implementation.
 
-    >>> df = spark.createDataFrame([('2015-04-08',)], ['a'])
-    >>> df.select(date_format('a', 'MM/dd/yyy').alias('date')).collect()
+    >>> df = spark.createDataFrame([('2015-04-08',)], ['time'])
```

Comment:
I am replacing all of the timestamps of form `yyyy-MM-dd` with `dt`, and all of the `yyyy-MM-dd HH:mm:ss` with ts, with some exceptions. Hopefully this addresses this point.